.babelrc

```diff
@@ -1,35 +1,50 @@
 {
-  presets: [
+  "presets": [
     [
-      "env",
+      "@babel/preset-env",
       {
-        targets: {
-          node: "current",
-          browsers: "last 2 chrome versions, last 2 firefox versions"
+        "targets": {
+          "node": "current",
+          "browsers": "last 2 chrome versions, last 2 firefox versions"
         },
-        modules: false
+        "modules": false
       }
     ],
-    "react",
-    "flow"
+    "@babel/preset-react",
+    "@babel/preset-flow"
   ],
-  plugins: [
-    "transform-class-properties",
-    [ "transform-object-rest-spread", { "useBuiltIns": true }]
+  "plugins": [
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
+    [
+      "@babel/plugin-proposal-object-rest-spread",
+      {
+        "useBuiltIns": true
+      }
+    ]
   ],
-  env: {
-    // Needed for tests, as node doesn't support ES2015 modules yet.
-    // The version of node we use doesn't need the generator polyfill so we're not including it
-    test: {
-      presets: [
-        [ "env", { target: { node: "current" }} ],
-        "react",
-        "flow"
-      ],
-      plugins: [
-        "transform-class-properties",
-        [ "transform-object-rest-spread", { "useBuiltIns": true }]
+  "env": {
+    "test": {
+      "presets": [
+        [
+          "@babel/preset-env",
+          {
+            "targets": {
+              "node": "current"
+            }
+          }
+        ],
+        "@babel/preset-react",
+        "@babel/preset-flow"
       ],
+      "plugins": [
+        ["@babel/plugin-proposal-class-properties", { loose: true }],
```

Comment:
"loose" was the default before. Without it we have +8kB in the result. The difference is that it uses `defineProperty` instead of direct assignments for these properties.